src/components/Button/Button.tsx

```diff
@@ -0,0 +1,23 @@
+import { FiDownload } from "react-icons/fi";
+import "./button.css";
+import { ButtonProps } from "./types";
+
+const Button = ({ type = "primary", disabled, children }: ButtonProps) => {
```

Comment:
most optimised way is 

import React, { PropsWithChildren } from 'react';

const Button = ({ type = 'primary', disabled, children }: PropsWithChildren<ButtonProps>) => {}

and don't have to add  children types in props type